ksql-engine/src/main/java/io/confluent/ksql/function/udf/datetime/CurrentTimestamp.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License; you may not use this file
+ * except in compliance with the License.  You may obtain a copy of the License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.function.udf.datetime;
+
+import io.confluent.ksql.function.KsqlFunctionException;
+import io.confluent.ksql.function.udf.Udf;
+import io.confluent.ksql.function.udf.UdfDescription;
+import io.confluent.ksql.function.udf.UdfParameter;
+import java.sql.Timestamp;
+import java.time.ZoneId;
+
+@UdfDescription(name = "currenttimestamp",
+    description = "Gets the current timestamp in milliseconds, represented as a BIGINT.")
+public class CurrentTimestamp {
+
+  @Udf(description = "Gets a BIGINT millisecond from the current timestamp.")
+  public long currentTimestamp() {
+    return new Timestamp(System.currentTimeMillis()).getTime();
```

Comment:
Why not just return `System.currentTimeMillis()`?

Reply:
Changed